crates/iroha_derive/tests/ui.rs

```diff
@@ -1,4 +1,3 @@
-#![cfg(not(coverage))]
```

Comment:
Accidental noise from local work - no need to remove this.

But it is harmful and was useless anyway.